site/docs/reference/Connectors/materialization-connectors/Rockset.md

```diff
@@ -42,4 +45,84 @@ materializations:
       	workspace: ${namespace_name}
       	collection: ${table_name}
     source: ${tenant}/${source_collection}
-```
\ No newline at end of file
+```
+## Bulk ingestion for large backfills of historical data
+
+You can backfill large amounts of historical data into Rockset using a *bulk ingestion*. Bulk ingestion must originate in S3 and requires additional steps in your dataflow. Flow's Rockset connector supports this through the GitOps workflow.
+
+### Prerequisites
+
+Before completing this workflow, make sure you have:
+* A working catalog spec including at least one Flow collection with its schema specified.
```

Comment:
nit: I think I'd remove "with its schema specified", since that's automatically the case if you have a collection.